production/ksonnet/loki/memcached.libsonnet

```diff
@@ -6,22 +6,38 @@ memcached {
     name: 'memcached',
     max_item_size: '2m',
     memory_limit_mb: 4096,
+
```

Comment:
This is better handled by the options exposed in the library, which also account for passing the args to memcached among other things:
https://github.com/grafana/jsonnet-libs/blob/master/memcached/memcached.libsonnet#L22-L31

Reply:
Ok, I am passing max_item_size and memory_limit_mb args to config.libsonnet